### PR TITLE
Add referral tracking to button

### PR DESCRIPTION
**Pull request by curiousjazz77**
Updated to be in line with that article, you sent Peter: https://flaviocopes.com/nextjs-server-client-code/

Moves forward our Airtable JIRA card: https://airtable.com/tblJy1oiCbCLjpTMr/viwV0wMqhSBakeaKS/recfkyNojgkqOyHLX
@pjstein

### CODE REVIEW

**Comment by curiousjazz77** [comment]
Note: troubleshooting the page error

File: pages/index.js [hunk 1]
```diff
@@ -281,8 +283,11 @@ function FaqList({ title, list }) {
   );
 }
 
+// const what = withRouter.query;
 // Main ------------------------------------------------------------------------
 export default function Home() {
+
+  const REFERRAL = typeof window !== 'undefined' ? `?prefill_Source=${encodeURIComponent(location.search)}` : "?prefill_Source=Louisiana%20Health%20Work%20Connect%20Website";
```
Comment (pjstein) [inline pages/index.js]: You want to extract the `source` parameter from the query string and use that as the `prefill_Source` parameter in the URL you're manufacturing. 

Right now, if say `https://healthworkconnect.la.gov/?this=is&a=test` is the URL, then `window.location.search` is `"?this=is&a=test"` and that is the value that will wind up prefilled in source. That won't be super useful on the backend.

File: pages/index.js [hunk 2]
```diff
@@ -281,8 +283,11 @@ function FaqList({ title, list }) {
   );
 }
 
+// const what = withRouter.query;
```
Comment (pjstein) [inline pages/index.js]: Delete.

**Review by pjstein — CHANGES_REQUESTED** [review]

**Comment by curiousjazz77** [comment]
last tested: https://la-health-work-connect-i7g1535n9.now.sh/?utm_campaign=dealofday&utm_source=linkedin&utm_medium=social-media

and 

https://la-health-work-connect-i7g1535n9.now.sh/?no_sourcing_added_test
https://la-health-work-connect-i7g1535n9.now.sh/?

**Review by pjstein — APPROVED** [review]

**Pull request merged**